draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1123,11 +1144,16 @@ An elliptic curve E is specified by an equation in two variables and a
 finite field F. An elliptic curve equation takes one of several standard forms,
```

Comment:
> ???
> Not clear that this is within the scope of the document, since right now we're not talking about the modulus M at all.

It might be good just to mention that there exists different irreducible polynomials to construct an extension field, even though they are isomorphic.

Reply:
Right now we just say "there is some basis", but we don't go into any more detail. I guess we could add something like "all choices of basis are isomorphic, but certain choices may result in a more efficient implementation", but it's not clear to me what this adds to the document...

Happy to add it if you think it's useful! :+1:

Reply:
Agreed -- noting the isomorphism, and perhaps reinforcing that this means the polynomial does not affect the basis, is sufficient.

Reply:
Cool. Added.